openc3/lib/openc3/api/tlm_api.rb

```diff
@@ -227,16 +227,16 @@ def get_tlm_buffer(target_name, packet_name, scope: $openc3_scope, token: $openc
     # @return [Array<String, Object, Symbol|nil>] Returns an Array consisting
     #   of [item name, item value, item limits state] where the item limits
     #   state can be one of {OpenC3::Limits::LIMITS_STATES}
-    def get_tlm_packet(target_name, packet_name, stale_time: 30, type: :CONVERTED, scope: $openc3_scope, token: $openc3_token)
+    def get_tlm_packet(target_name, packet_name, stale_time: 30, type: :CONVERTED, cache_timeout: 0.1, scope: $openc3_scope, token: $openc3_token)
       target_name = target_name.upcase
       packet_name = packet_name.upcase
       authorize(permission: 'tlm', target_name: target_name, packet_name: packet_name, scope: scope, token: token)
       packet = TargetModel.packet(target_name, packet_name, scope: scope)
       t = _validate_tlm_type(type)
       raise ArgumentError, "Unknown type '#{type}' for #{target_name} #{packet_name}" if t.nil?
       items = packet['items'].map { | item | item['name'].upcase }
-      cvt_items = items.map { | item | "#{target_name}__#{packet_name}__#{item}__#{type}" }
-      current_values = CvtModel.get_tlm_values(cvt_items, stale_time: stale_time, scope: scope)
+      cvt_items = items.map { | item | [target_name, packet_name, item, type] }
+      current_values = CvtModel.get_tlm_values(cvt_items, stale_time: stale_time, cache_timeout: cache_timeout, scope: scope)
```

Comment:
Are you changing this method to take an array of array of strings? It used to be an array of strings with the underscores.

Reply:
Yes, this was just an optimization to prevent splitting twice.

Reply:
Won't that break existing code?

Reply:
If anyone is using CvtModel.get_tlm_values this is a breaking change.  That's an internal api though.